src/avx/oapv_tq_avx.c

```diff
@@ -95,10 +95,95 @@ static void oapv_tx_part_avx(s16 *src, s16 *dst, int shift, int line)
     _mm_store_si128((__m128i *)(dst + 6 * line), _mm256_castsi256_si128(d1));
     _mm_store_si128((__m128i *)(dst + 7 * line), _mm256_extracti128_si256(d1, 1));
 }
+const oapv_fn_tx_part_t oapv_tbl_fn_txb_part_avx[2] =
+{
+    oapv_tx_part_avx,
+        NULL
+};
+
```

Comment:
I don't know whether this structure is actually used.
fn_tx_part_t variable is not called anwhere...